Parsers/ASimNetworkSession/Parsers/vimNetworkSessionMicrosoftMD4IoT.yaml

```diff
@@ -117,11 +138,24 @@ ParserQuery: |
       DirectionNetworkEvents
       | where not(outbound)
       // ***************  Postfilterring *****************************************************************
-      | where (array_length(srcipaddr_has_any_prefix)==0 or RemoteAddress==srcipaddr_has_any_prefix) and
-              (array_length(dstipaddr_has_any_prefix)==0 or LocalAddress ==dstipaddr_has_any_prefix) and
-              (isnull(dstportnumber) or dstportnumber==RemotePort)
+      | where (isnull(dstportnumber) or dstportnumber==RemotePort)
       // ***************  Postfilterring *****************************************************************
       | invoke parser ()
+      | extend temp_isSrcMatch=( // only one of each pair has_any_ipv4_prefix is calculated
```

Comment:
I think here you still pre-filter after parsing.

Reply:
yes. I need the parser to extract the remote/local ip